multicore_runtime/cloud_logging_test.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import json
+import logging
+import mock
+import os
+import unittest
+
+import cloud_logging
```

Comment:
No, it's not a blocker.

I think the recommendation is still to use absolute, but I agree that's easier said than done, particularly outside of the internal repositories that impose a directory structure suitable for it...

I do think we should go with the explicit relative imports (`from . import cloud_logging`) rather than the implicit ones (`import cloud_logging`) because I *do* think there's a lot wrong with the latter, they could be shadowed by other modules with the same name at the top-level, etc. (As mentioned, Python 3 removed this "feature".) So, if you can implement that without breaking anything (sometimes there's some non-obvious places where you have dependencies) then please go for it. If it does break it, please give me more details so we can discuss that, at the very least I think we should add a comment indicating why it's being done this way...